one-ui/ui/src/config/data.config.js

```diff
@@ -52,10 +61,14 @@ const mapProps = {
       name: { sourcedFrom: 'name' }
     }
   },
+  namespaces: {
+    nutFree: 'http://namespaces/nutfree',
+    withNuts: 'http://namespaces/withNuts'
+  },
   mapName: 'testMap',
   getMappingArtifactByMapName: jest.fn(),
   updateMappingArtifact: jest.fn(),
-  mappingVisible:  false,
+  mappingVisible:  true,
```

Comment:
hope this doesnt break any other tests which requires mappingVisible to be false. We can always render with props overridden from this config file.. Like below so we can keep data.config generic:
render(<SourceToEntityMap {...data.mapProps} mappingVisible={true}/>)

Reply:
Updated the test. Please review now.